interface/reports/patient_edu_web_lookup.php

```diff
@@ -55,76 +42,53 @@ function searchResultsPopup(search_term,link) {
   }
 </script>
 
-<title><?php echo htmlspecialchars( xl('Find Patient Education Materials'), ENT_NOQUOTES); ?></title>
 </head>
 
-<body class="body_top">
-
-<span class='title'><?php echo htmlspecialchars( xl('Web Search'), ENT_NOQUOTES); ?> - <?php echo htmlspecialchars( xl('Patient Education Materials'), ENT_NOQUOTES); ?></span>
-
+<body class="body_top" onload="javascripts:document.forms[0].form_import_data.focus()">
+<div class="container">
```

Comment:
When I removed one container div the text "Search in "  does not show the "S". The two container div works better.